packages/css/src/components/select/select.scss

```diff
@@ -4,7 +4,7 @@
  */
 
 @mixin reset {
-  appearance: none;
+  appearance: none; // Reset native appearance to hide default chevron
```

Comment:
Why not a webkit prefix here?

Reply:
I guess because it's not an input. Yeah.

Reply:
`appearance: none` is used here to hide the default chevron. That isn't necessary for Safari < 15.4 (which are the browsers for which we sometimes need the prefix)